src/System.Net.Requests/src/System/Net/FtpWebRequest.cs

```diff
@@ -1576,7 +1576,7 @@ public bool KeepAlive
         {
             get
             {
-                return false;
```

Comment:
That doesn't actually answer my question.

The property is documented here: https://msdn.microsoft.com/en-us/library/system.net.ftpwebrequest.keepalive(v=vs.110).aspx
If the new default is "true" will CoreFX behave as documented?